### PR TITLE
Feature/refactorpredicates

### LINKED ISSUE #45: Refacto moreThan to greaterThan
- [x] I have read the [CONTRIBUTING](https://github.com/leoture/MockSwift/blob/master/CONTRIBUTING.md).  

## 🚀 Describe the feature you'd like
Change predicates: 
- `moreThan` to `greaterThan`
- `moreThanOrEqualsTo` to `greaterThanOrEqualsTo`

### PR DESCRIPTION
**Pull request by parisempress**
>**⚠️ IMPORTANT: Please do not create a Pull Request without creating an issue first. ⚠️**  
Please provide enough information so that others can review your pull request.  
**REMOVE THIS SECTION**

---

- [ ] I have read the [CONTRIBUTING](https://github.com/leoture/MockSwift/blob/master/CONTRIBUTING.md).  

###### 🔗 Closing issues  
This closes #45 

# 📖 Summary

### CODE REVIEW

**Comment by leoture** [comment]
🎉 Thank you for your contribution! Unfortunately, some tests do not pass. Make sure all tests are successful before creating a PR. You can use the Makefile defined in the project.

File: Sources/MockSwift/Predicates/Predicate+Comparable.swift [hunk 1]
```diff
@@ -46,30 +46,30 @@ prefix operator >
...
+    Predicate.match(description: "greater than \(rhs)") { $0 > rhs }
   }
 
   /// Creates a `Predicate<Input>` that can match any elements more than or equal to `rhs`
   /// - Parameter rhs: The element to compare to.
   /// - SeeAlso: `Predicate`
-  static func moreThanOrEqualsTo(rhs: Input) -> Predicate<Input> {
-    Predicate.match(description: "more than or equals to \(rhs)") { $0 >= rhs }
+  static func greaterThanOrEqualsTo(rhs: Input) -> Predicate<Input> {
+    Predicate.match(description: "greaterThanOrEqualsTo \(rhs)") { $0 >= rhs }
```
Comment (leoture) [inline Sources/MockSwift/Predicates/Predicate+Comparable.swift]: `"greater than or equals to \(rhs)"`

Comment (parisempress) [reply Sources/MockSwift/Predicates/Predicate+Comparable.swift]: Thank you for your input, I will review and make changes

Comment (leoture) [reply Sources/MockSwift/Predicates/Predicate+Comparable.swift]: Hello! I'm waiting your changes before to merge your branch. 🙂 Don't forget to rebase your branch and run tests with fastlane.

Comment (leoture) [reply Sources/MockSwift/Predicates/Predicate+Comparable.swift]: I close this PR because #85 was merged.

**Pull request closed without merging**